src/components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import "./ProductCard.css";
+
+const ProductCard = ({
+  cardImage,
+  cardTitle,
+  cardBadge,
+  cardPrice,
+  cardPriceBefore,
+  cardDiscount,
```

Comment:
As we know that this is the data for a card then we could avoid prefixing `card` before all the props and could simply write `image`, `title`, etc.

Reply:
Thanks Vishal for the reveiw, Will look into the suggestions...